code/modules/mafia/controller.dm

```diff
@@ -192,6 +193,14 @@
 	var/datum/mafia_role/loser = get_vote_winner("Day")//, majority_of_town = TRUE)
 	if(loser)
 		send_message("<b>[loser.body.real_name] wins the day vote, Listen to their defense and vote \"INNOCENT\" or \"GUILTY\"!</b>")
+		//refresh the lists
+		judgement_abstain_votes = list()
+		judgement_innocent_votes = list()
+		judgement_guilty_votes = list()
+		for(var/i in all_roles)
+			var/datum/mafia_role/abstainee = i
```

Comment:
you can do the same for the loops below, don't really see why you wouldn't if you know they're all the correct type

```suggestion
		for(var/datum/mafia_role/abstainee in all_roles)
```

Reply:
faster to not use byond's type filtering feature (if negligable)